LinkedList.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Created by ioana-chirca on 16-May-17.
+ */
+public class LinkedList {
+    public class Node {
+        private int value;
+        private Node next;
+
+        public Node() {}
+
+        public Node(int value) {
+            this.value = value;
+        }
+
+        public void setValue(int value) {
+            this.value = value;
+        }
+
+        public int getValue() {
+            return value;
+        }
+
+        public void setNext(Node next) {
+            this.next = next;
+        }
+
+        public Node getNext() {
+            return next;
+        }
+    }
+
+    private Node head;
+    private int size;
+
+    public LinkedList() {
+    }
+
+    public int getSize() {
+        return size;
+    }
+
+    public void addLast(int value) {
```

Comment:
Right now the complexity of this method is O(n) because I'm always traversing the list. I guess I could add a Node in O(1) if I'm adding to the front of the list or if I keep a pointer to the last node.